sessions/using-delay-distributions-to-model-the-data-generating-process-of-an-epidemic.qmd

```diff
@@ -74,12 +74,25 @@ Now we can convolve the time series with a delay distribution to get a time seri
 
 In our first session, we decided to assume the delay from infection to symptom onset had a gamma distribution.
 However, if we want to use the gamma distribution with shape 5 and rate 1 as before, we face a familiar issue.
-The gamma distribution is a *continuous* distribution, but now our delay data are in days which are *discrete* entities.
-Before we can simulate a time series of symptom onsets, we will first need to generate a discrete probability mass function of the incubation period (in days).
+The gamma distribution is a *continuous* distribution, but now our delay data are in days which are *discrete* entities. In addition to this, we are assuming that both events that make up the delay (here infections and onsets) are observed as daily counts.
+This means that our distribution is **double interval censored** which we encountered in the [the biases in delay distribution session](sessions/biases-in-delay-distributions.qmd), so we need to use the same ideas introduced in that session. The important point to remember is that the cumulative distribution function ($F(t)$) of a distribution that has a daily censored primary event can be expressed as,
+$$
+F^*(t) = \int_0^1 F(t - u) du
+$$
+
+The probability mass function (PMF) is then
+
+$$
+f_t \propto \frac{F^*(t + 1) - F^*(t - 1)}{}
+$$
 
+which you may be familiar with from single censored (i.e. for the secondary event) discretised distributions.
```

Comment:
this is assuming we communicated this in the biases in delays session. If we aren't I think we should

Reply:
we only discuss double censored there, and no maths

Reply:
So I think the options are to take out the maths here or leave?

As it was we had some maths but it wasn't really correct so that doesn't seem like a good option?

Reply:
I agree - this might be from before we decided to use simulation. I'll update.

Reply:
as an aside this is what we need to add to `EpiNow2` + elsewhere. you solve the first integral numerically and the do the second thing as normal. stan support for auto-diffing through numerical integrals should be good so this should work pretty well